morepath/tests/test_json_obj.py

```diff
@@ -25,6 +25,30 @@ def dump_model_json(self, request):
     assert response.json == {'x': 'foo'}
 
 
+def test_json_obj_dump_app_arg():
+    class App(morepath.App):
+        pass
+
+    @App.path(path='/models/{x}')
+    class Model(object):
+        def __init__(self, x):
+            self.x = x
+
+    @App.json(model=Model)
+    def default(self, request):
+        return self
+
+    @App.dump_json(model=Model)
+    def dump_model_json(app, self, request):
```

Comment:
You're not proposing to change this in views as well, correct? A long time ago I used to call it ``model``, but I switched to ``self`` as it helps with people's intuitions. Plus the confusion between class and instance -- does model refer to a class or to an instance? 

See this discussion here:

https://github.com/morepath/morepath/issues/101

This is why I'd be inclined to call it ``obj`` instead of ``model`` in the arguments, even though it doesn't match the predicate.